.gitignore

```
@@ -37,3 +37,6 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+# nx
+.nx
```

I know I suggested this but I think it should actually be

```suggestion
.nx/cache
```

based on https://github.com/microsoft/fluentui/blob/master/.gitignore#L161

@Hotell should be able to confirm

---

nx migration did this, I trust robots ;) :D lets keep it as is